### ISSUE TITLE
Webhooks access issue

### ISSUE THREAD

**Issue by kamieniarz** [WatWowMap/ReactMap#353]
tldr; Poracle role without access to `monster` doesn't have the access to webhooks if it doesn't have the access to `perms` -> `pokemon`

If poracle has enabled `commandSecurity` and the role isn't listed in `monster` field (so basically that role won't be notified about any mons) and also it's not listed in reactmap config `perms` -> `pokemon` (which is pretty obvious cause that specific role won't have access to mons in poracle and on map) webhooks won't work. Users with that role will see `non_registered_human_desc` message.
I have tested `iv` and `pvp` in `perms` and looks like these are not the blocker in that case.

**Comment by clburlison** [comment]
Is this still a valid enhancement? Not a situation I run into on my map.

**Comment by kamieniarz** [comment]
This issue still exists

**Comment by Jebula999** [comment]
I just ran into this issue as well.
I have different levels of access, and some access only grant raid information.

Happened upon this issue when trying to allow the users to use the alerts feature while they only have access to raid data.

**Comment by kamieniarz** [comment]
Fixed in https://github.com/WatWowMap/ReactMap/pull/755 😄
